content/en/docs/reference/kubectl/jsonpath.md

```diff
@@ -92,8 +92,8 @@ kubectl get pods -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.status.st
 On Windows, you must _double_ quote any JSONPath template that contains spaces (not single quote as shown above for bash). This in turn means that you must use a single quote or escaped double quote around any literals in the template. For example:
 
 ```cmd
-C:\> kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{'\t'}{.status.startTime}{'\n'}{end}"
-C:\> kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{\"\t\"}{.status.startTime}{\"\n\"}{end}"
+kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{'\t'}{.status.startTime}{'\n'}{end}"
```

Comment:
Hello @merkata . If I recall correctly, lines 94 - 95 were formatted explicitly for `Windows` (hence the C:\ prompt).
Did the prompt help indicate that the command is targeted for Windows users or does the description on line 92 provide enough information?

Reply:
This is due to #18853 and the comment https://github.com/kubernetes/website/pull/18853#discussion_r372823426 from @sftim

Reply:
@merkata , Yes, the style guide recommends to exclude a command prompt.
To emphasize that this section applies to Windows users, I am wondering if lines 92 - 96 should be enclosed
in a note shortcode?
```
{{< note >}}

{{< /note >}}
```

Reply:
Is it possible to write one sample command that works on both POSIX / Bourne shell and Windows command shell (maybe PowerShell too?)

Reply:
@kbhawkey good point, I've pushed the change, thanks!